pom.xml

```diff
@@ -18,7 +18,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <!-- please follow the instructions in 'updatingDependencies.md' when upgrading WildFly -->
-        <wildfly.version>31.0.1.Final</wildfly.version>
+        <wildfly.version>32.0.1.Final</wildfly.version>
```

Comment:
Do we need to bump wildfly dependencies in gradle?

Reply:
Yes I did. :-)